STYLE.md

```diff
@@ -131,15 +131,188 @@ if status is not pebble.ServiceStatus.ACTIVE:
+
+Examples:
+
+- Avoid: "This can be done easily using ..."
+- Prefer: "This can be done using ..."
+
+- Avoid: "This can be easily configured by ..."
+- Prefer: "We can configure this using ..."
+
+- Avoid: "Simply run the command ..."
+- Prefer: "Run the command ..."
+
+### Structure
+
+#### Be short
+
+For example, a very long introduction part of a how-to guide or a tutorial probably isn't necessary; it's good if we are writing a blog post or a book, but not so much for a tutorial/how-to guide type of doc.
+
+#### Be concise and reduce repetition
+
+Trim repetitive titles: Avoid repeating the name if the surrounding content already provides context.
+
+For example, in the doc "how to use pebble", the section names can simply be "API", "CLI commands", "Features", etc., instead of "Pebble API", "Pebble CLI commands", "Pebble features".
+
+#### Order: alphanumeric or logical
+
+We can either use an alphanumeric order, or a logical order, depending on the context.
+
+Alphanumeric order is better in the case of a reference doc or a list (e.g., an Enum).
+
+Example (Enum):
+
+- Prefer: `["autostart", "replan", "restart", "start", "stop"]`
+- Avoid: `["start", "stop", "restart", "replan", "autostart"]`
+
+Example (A list of commands):
+
+Avoid:
+
+- `pebble ls`
+- `pebble mkdir`
+- `pebble rm`
+- `pebble push`
+- `pebble pull`
+
+Prefer:
+
+- `pebble ls`
+- `pebble mkdir`
+- `pebble pull`
+- `pebble push`
+- `pebble rm`
+
+Exceptions:
+
+- If there's a strong logical progression (e.g., "low", "medium", "high"; or a list of things that you should read in that particular order).
+- If there is already an exception in the current documentation, follow existing conventions.
+
+### Content
+
+#### Use British English
 
 [Canonical's documentation style](https://docs.ubuntu.com/styleguide/en/) uses British spelling, which we try to follow here. For example: "colour" rather than "color", "labelled" rather than "labeled", "serialise" rather than "serialize", and so on.
 
 It's a bit less clear when we're dealing with code and APIs, as those normally use US English, for example, `pytest.mark.parametrize`, and `color: #fff`.
 
+#### Spell out abbreviations
 
-### Spell out abbreviations
+Abbreviations and acronyms in docstrings should usually be spelled out:
 
-Abbreviations and acronyms in docstrings should usually be spelled out, for example, "for example" rather than "e.g.", "that is" rather than "i.e.", "and so on" rather than "etc", and "unit testing" rather than UT.
+- "for example" rather than "e.g."
+- "that is" rather than "i.e."
+- "and so on" rather than "etc"
+- "unit testing" rather than UT, and so on
 
 However, it's okay to use acronyms that are very well known in our domain, like HTTP or JSON or RPC.
+
+#### Use sentence case in headings
+
+`## Use sentence case for headings`, instead of `## Use Title Case for Headings`.
+
+#### Be consistent
+
+Be consistent in every possible way as much as possible, and stay context-consistent.
```

Comment:
This is a funny redundant sentence "in every possible way" and "as much as possible". What about just dropping it?